test/gutenberg-test-themes/behaviors-false/style.css

```diff
@@ -0,0 +1,15 @@
+/*
+Theme Name: Behaviors
```

Comment:
I know these are test themes and should be independent, but could it help to give them distinct names that correspond to their use? Even `Behaviors (false)` might give clues when debugging that differentiate it from the other `Behaviors` themes.